spec/factories.rb

```diff
@@ -362,6 +362,7 @@
     association :investment, factory: :budget_investment
     sequence(:title)     { |n| "Budget investment milestone #{n} title" }
     description          'Milestone description'
+    publication_date     Time.zone.today
```

Comment:
I assume this `Time.zone.today` works but `Time.zone.today.to_date` would be better?

Reply:
As far as I know, it would make no difference. `to.date` is only necessary for Capybara tests.